packages/seacas/applications/blot/cnvmax.f

```diff
@@ -48,7 +48,7 @@ SUBROUTINE CNVMAX (NUMNPF, VARNP, IN2ELB, NMIN, NMAX, FMIN, FMAX)
 C   --   Written by Amy Gilkey - revised 03/09/88
```

Comment:
Let no one forget that our software has a long lifetime :-)

Reply:
Yes, older than some of the people running it...